project/BuildHelper.scala

```diff
@@ -6,61 +6,8 @@ import BuildInfoKeys._
 object BuildHelper {
   private val Scala212             = "2.12.14"
   private val Scala213             = "2.13.8"
-  private val SilencerVersion      = "1.7.8"
-  private val KindProjectorVersion = "0.10.3"
-
-  private val stdOptions = Seq(
```

Comment:
I wouldn't remove these options as they are valid for scala 2.x

Reply:
I removed these in favor of `tpolecatplugin` https://github.com/typelevel/sbt-tpolecat, because I'm not sure which ones to keep for Scala 3 😬